openqa-schedule-mm-ping-test

```diff
@@ -53,7 +54,8 @@ job_templates:
       PARALLEL_WITH: ping_server
 EOF
 
-hdd=$(runcli openqa-cli api --host "$openqa_url" jobs version="$version" scope=relevant arch="$arch" flavor="$flavor" test="$test_name" latest=1 | runjq -r '.jobs | map(select(.result == "passed")) | max_by(.settings.BUILD) .settings.HDD_1')
+jobs=$(runcli openqa-cli api --host "$openqa_url" jobs version="$version" scope=relevant arch="$arch" flavor="$flavor" test="$test_name" latest=1 | runjq -r .jobs)
+hdd=$(echo $jobs | runjq -r "map(select(.result | test(\"(${allowed_results})\"))) | max_by(.settings.BUILD) .settings.HDD_1")
 time openqa-cli schedule \
```

Comment:
I guess this call is the problematic one.

Reply:
Right. I'm looking into extending the monitor command now: https://github.com/os-autoinst/openQA/pull/6101